ecs-init/docker/docker_config.go

```diff
@@ -97,3 +98,24 @@ func getNsenterBinds(statFn func(string) (os.FileInfo, error)) []string {
 	}
 	return binds
 }
+
+// Returns modinfo bind as a slice if modinfo is available on the host.
+// Otherwise, it will return an empty slice.
+func getModInfoBinds(statFn func(string) (os.FileInfo, error)) []string {
+	binds := []string{}
+	modInfoPathLocations := []string{
+		"/sbin/modinfo",
+		"/usr/sbin/modinfo",
+	}
+	for _, path := range modInfoPathLocations {
+		if _, err := statFn(path); err == nil {
+			seelog.Debugf("modinfo found at %s", path)
+			binds = append(binds, path+":"+path)
+			break
+		} else {
+			seelog.Warnf("modinfo not found at %s, skip binding it to Agent container: %v",
```

Comment:
nit: I think an INFO level log should be fine since we are trying two paths?

Reply:
that's fair